src/tests/system/tests/test_access_control.py

```diff
@@ -0,0 +1,139 @@
+"""
+SSSD Authentication Test Cases
+
+:requirement: access control
+"""
+
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopologyGroup
+
+
+@pytest.mark.topology(KnownTopologyGroup.AnyProvider)
+@pytest.mark.parametrize("sssd_service_user", ("root", "sssd"))
+@pytest.mark.parametrize("method", ["su", "ssh"])
+@pytest.mark.importance("critical")
+@pytest.mark.require(
+    lambda client, sssd_service_user: ((sssd_service_user == "root") or client.features["non-privileged"]),
+    "SSSD was built without support for running under non-root",
+)
+def test_access_control__simple_filter_permits_user_login(
+    client: Client, provider: GenericProvider, method: str, sssd_service_user: str
+):
+    """
+    :title: Simple access filter permits user login
+    :setup:
+        1. Create users ‘user1’ and ‘user2’
+        2. Configure SSSD with ‘access_provider = simple’, ‘simple_allow_users = user1’
+        3. Start SSSD
+    :steps:
+        1. Try to login as ‘user1’
+        2. Try to login as ‘user2’
+    :expectedresults:
+        1. User1 can login
+        2. User2 cannot login
+    :customerscenario: False
+    """
+    provider.user("user1").add(password="Secret123")
+    provider.user("user2").add(password="Secret123")
+
+    client.sssd.domain["access_provider"] = "simple"
+    client.sssd.domain["simple_allow_users"] = "user1"
+
+    client.sssd.start(service_user=sssd_service_user)
+
+    assert client.auth.parametrize(method).password("user1", "Secret123"), "User login!"
```

Comment:
```client.auth.ssh.password(...)```

The second part, is the stderr message, if the assertion fails. Should be the negation, ```User cannot login!"

Reply:
So all the messages afterwards the assertion should be the opposite? Is that right? In this case I will change all of them, because I understood it wrong ;)

Reply:
I liked more this style I implemented in the last test case:

>    assert client.auth.ssh.password("user2", "Secret123"), "User2 should be able to log in!"
>    assert not client.auth.ssh.password("user3", "Secret123"), "User3 should NOT be able to log in!"

wdyt?